site/docs/id/guide/deployment-types.md

```diff
@@ -236,6 +236,8 @@ app.use(webhookCallback(bot, "oak"));
 :::
 ::::
 
+> Kamu tidak perlu memanggil `bot.start()` ketika menggunakan webhooks.
```

Comment:
```suggestion
> Kamu tidak perlu memanggil `bot.start()` ketika menggunakan webhook.
```

Do we still need `s` in bahasa?

Reply:
I am not sure though. What do you think?

Reply:
@fadzikri?

Reply:
I think both of them is okay. But for the shake of consistency, we should determine which one should be used. This one is safe to be merged.